types/src/model/mod.rs

```diff
@@ -65,3 +69,27 @@ pub use self::{
         UnloadWallet, WalletCreateFundedPsbt, WalletDisplayAddress, WalletProcessPsbt,
     },
 };
+
+/// Models the data returned by Core for a scriptPubKey.
+///
+/// This is used by methods in the blockchain section and in the raw transaction section (i.e raw
+/// transaction and psbt methods). The shape changed in Core v22 but the new shape is fully
+/// backwards compatible so we only provide it not a v0.17 specific type.
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+#[cfg_attr(feature = "serde-deny-unknown-fields", serde(deny_unknown_fields))]
+pub struct ScriptPubkey {
+    /// The script_pubkey parsed from hex.
+    pub script_pubkey: ScriptBuf,
+    /// Number of required signatures - deprecated in Core v22.
+    ///
+    /// Only returned before in versions prior to 22 or for version 22 onwards if
+    /// config option `-deprecatedrpc=addresses` is passed.
```

Comment:
```suggestion
    /// Only returned in versions prior to 22 or for version 22 onwards if
    /// config option `-deprecatedrpc=addresses` is passed.
```

Reply:
There are 4 instances of this typo in the codebase:

```
gg 'returned before in'
lib.rs:196:    /// Only returned before in versions prior to 22 or for version 22 onwards if
lib.rs:207:    /// Only returned before in versions prior to 22 or for version 22 onwards if
model/mod.rs:85:    /// Only returned before in versions prior to 22 or for version 22 onwards if
model/mod.rs:92:    /// Only returned before in versions prior to 22 or for version 22 onwards if
```

Reply:
Fixed.